examples/src/main/scala/org/apache/spark/examples/snappydata/WorkingWithJson.scala

```diff
@@ -102,6 +102,38 @@ object WorkingWithJson extends SnappySQLJob {
 
     val allPersons = nameAndAddress.toJSON
     allPersons.show(truncate = false)
+
+    // Multi-line json example
+    println("Multi-line json example\n################")
+    val locomotives_path = s"${jobConfig.getString("json_resource_folder")}/locomotives.json"
+    val locomotivesDF = snSession.read.option("wholeFile", "true").json(locomotives_path)
+    println("Printing schema of the DataFrame locomotivesDF")
+    locomotivesDF.printSchema()
+    println("DataFrame locomotivesDF contents\n################")
+    locomotivesDF.show(false)
+    println(s"locomotivesDF row count: ${locomotivesDF.count()}")
+
+    snSession.dropTable("locomotives", ifExists = true)
+    snSession.createTable(tableName = "locomotives",
+      provider = "column",
+      schema = locomotivesDF.schema,
+      options = Map.empty[String, String],
+      allowExisting = false)
+    locomotivesDF.write.insertInto("locomotives")
+
+    val resultDF = snSession.sql("SELECT dateIso, emission_tier, engine, fleet" +
+        ", hqLatLng, installedOn, manufacturer, model," +
+        " parent, serial_no, type, uri FROM locomotives")
+    println("Contents of the table locomotives:\n################")
+    resultDF.show(truncate = false)
+
+    snSession.sql("CREATE TABLE stg_locomotives (dateIso STRING, emission_tier STRING, engine " +
```

Comment:
SQL should be "CREATE EXTERNAL TABLE....", i.e., EXTERNAL keyword should be added.

Reply:
Noted.